templates/todo/projects/nodejs-mongo/.repo/terraform/infra/main.tf

```diff
@@ -24,11 +25,110 @@ resource "azurerm_resource_group" "rg" {
 # Deploy application insights
 # ------------------------------------------------------------------------------------------------------
 module "applicationinsights" {
-  source           = "../../../../../../common/infra/terraform/applicationinsights"
+  source           = "../../../../../../common/infra/terraform/core/monitor/applicationinsights"
   location         = var.location
   rg_name          = azurerm_resource_group.rg.name
   environment_name = var.environment_name
-  workspace_id     = azurerm_log_analytics_workspace.workspace.id
+  workspace_id     = module.loganalytics.LOGANALYTICS_WORKSPACE_ID
   tags             = azurerm_resource_group.rg.tags
   resource_token   = local.resource_token
 }
+
+# ------------------------------------------------------------------------------------------------------
+# Deploy log analytics
+# ------------------------------------------------------------------------------------------------------
+module "loganalytics" {

```

Comment:
In bicep we have a single monitoring.bicep that includes both app insights and log analytics

Reply:
resolving it after an offline discussion in which we decided to keep the modules as is.